src/jit/compiler.h

```diff
@@ -487,6 +487,45 @@ class LclVarDsc
         assert(_lvRegNum == reg);
     }
 
+    const size_t lvArgStackSize() const
+    {
```

Comment:
This is also pretty big for a header implementation. Can you move the implementation to a cpp file?

---
In reply to: [133322034](https://github.com/dotnet/coreclr/pull/12489#discussion_r133322034) [](ancestors = 133322034)